src/tests/CullingTest.cpp

```diff
@@ -119,7 +136,7 @@ TEST(CullingTest, frustumCulling) {
       esp::assets::AssetInfo::fromPath(sceneFile);
   bool loadSuccess =
       resourceManager.loadScene(info, &sceneRootNode, &drawables);
-  EXPECT_EQ(loadSuccess, true);
+  CORRADE_COMPARE(loadSuccess, true);
```

Comment:
```suggestion
  CORRADE_VERIFY(resourceManager.loadScene(info, &sceneRootNode, &drawables));
```

(oh, cool, multi-line suggestions finally (at least partially) work? :O)